src/rotary/rotary_fti.c

```diff
@@ -27,6 +27,11 @@
 #include "rotary.h"
 #include "upm_fti.h"
 
+// M_PI is not a C standard (C11)
+#ifndef M_PI
+#define M_PI           3.14159265358979323846
+#endif
+
```

Comment:
This would go into `upm_math.h` as it's used by multiple sensors, exactly why I added the file for 😉.